python/hsfs/feature_view.py

```diff
@@ -0,0 +1,590 @@
+#
+#   Copyright 2022 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import warnings
+from datetime import datetime
+from typing import Optional, Union, List, Dict, Any
+
+import humps
+
+from hsfs import (
+    tag, util, training_dataset_feature, storage_connector, training_dataset,
+)
+from hsfs.constructor import query
+from hsfs.core import (
+    feature_view_engine,
+    transformation_function_engine, vector_server
+)
+from hsfs.transformation_function import TransformationFunction
+from hsfs.statistics_config import StatisticsConfig
+
+
+class FeatureView:
+    ENTITY_TYPE = "featureview"
+
+    def __init__(
+        self,
+        name: str,
+        query,
+        featurestore_id,
+        id=None,
+        version: Optional[int] = None,
+        description: Optional[str] = "",
+        label: Optional[List[str]] = [],
```

Comment:
you removed the label from the feature view in the hopsworks, i guess it is not needed here any more.

Reply:
We still have label in feature view but it is saved inside `Feature` but not as a separate field.